tests/gs1/test_messages.py

```diff
@@ -295,6 +296,73 @@ def test_parse_hri(value: str, expected: GS1Message) -> None:
     assert GS1Message.parse_hri(value) == expected
 
 
+def test_parse_hri_with_gtin_with_variable_weight() -> None:
+    result = GS1Message.parse_hri(
+        "(01)02302148210869",
+        rcn_region=RcnRegion.NORWAY,
```

Comment:
I'm still reading up on this.

Reply:
It's not much to worry about really. The `parse_hri()` method just needs to be able to pass the RCN region through to the GS1 parser, so that it can interpret the RCN-subset of GTIN with the local interpretation rules.